srtcore/srt.h

```diff
@@ -573,9 +573,17 @@ enum SRT_REJECT_REASON
     SRT_REJ_FILTER,      // incompatible packet filter
     SRT_REJ_GROUP,       // incompatible group
 
-    SRT_REJ__SIZE,
+    SRT_REJ_E_SIZE,
```

Comment:
API change. Someone could have already used `SRT_REJ__SIZE`.

Reply:
I'll make a `#define`. This label must be deprecated - symbols with double `_` inside are reserved for the compiler - I've learned that quite late.